Makefile

```diff
@@ -20,15 +20,16 @@ UNAME_S := $(shell uname -s)
 USERID  := $(shell id -u)
 GRPID   := $(shell id -g)
 
-.PHONY: bin test-unit test-integration
+.PHONY: bin
+bin:
+	@mkdir -p $(BINDIR)
+	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' github.com/vmware-tanzu/antrea/cmd/...
```

Comment:
I remember this not working with older versions of go (`go build -o <dir>` with multiple executables). It's great that they fixed it in go 1.13!

Reply:
Yeah, it is fixed by this https://github.com/golang/go/commit/b48bda9c6f57ca9a940eac95700485b9640a62e9.